Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -576,6 +576,46 @@ public bool TryParseInterface(string token, out Collection<IPObject>? result)
             return false;
         }
 
+        /// <summary>
+        /// Checks to see if <paramref name="remoteIp"/> has access.
+        /// </summary>
+        /// <param name="remoteIp">IP Address of client.</param>
+        /// <returns><b>True</b> if has access, otherwise <b>false</b>.</returns>
+        public bool HasRemoteAccess(IPAddress remoteIp)
+        {
+            var config = _configurationManager.GetNetworkConfiguration();
+            if (config.EnableRemoteAccess)
+            {
+                // Comma separated list of IP addresses or IP/netmask entries for networks that will be allowed to connect remotely.
+                // If left blank, all remote addresses will be allowed.
+                var remoteAddressFilter = RemoteAddressFilter;
+
+                if (RemoteAddressFilter.Count > 0 && !IsInLocalNetwork(remoteIp))
+                {
+                    // remoteAddressFilter is a whitelist or blacklist.
+                    bool isListed = RemoteAddressFilter.ContainsAddress(remoteIp);
+                    if (config.IsRemoteIPFilterBlacklist)
+                    {
+                        // Black list, so flip over.
+                        isListed = !isListed;
+                    }
+
+                    if (!isListed)
+                    {
+                        // If your name isn't on the list, you arn't coming in.
+                        return false;
+                    }
```

Comment:
This should be the same I think. Not sure if it's more confusing though.
```suggestion
                    return RemoteAddressFilter.ContainsAddress(remoteIp) == !config.IsRemoteIPFilterBlacklist;
```

Reply:
Can't be any more confusing than the original. Don't know how many time's i've scratched my head with it.